inductiva/_cli/cmd_tasks/kill.py

```diff
@@ -1,20 +1,49 @@
 """Kills a tasks by id via CLI."""
+import sys
 
 import inductiva
+from inductiva import constants
+from inductiva.tasks.methods import get_all
+from inductiva.utils.input_functions import user_confirmation_prompt
+from ...localization import translator as __
+
+#             "pending-input": "PENDINGINPUT",
+#             "submitted": "SUBMITTED",
+#             "started": "STARTED",
+#             "pending-kill": "PENDINGKILL",
+#             "zombie": "ZOMBIE",
 
 
 def kill_task(args):
     """Kills a task by id."""
-    if not args.yes:
-        prompt = input("Do you confirm you want to "
-                       f"kill {len(args.id)} tasks (y/[N])?")
-        confirm = prompt.lower() in ["y", "ye", "yes"]
-        if not confirm:
-            print("Aborted.")
-            return
+    kill_all = args.all
+    ids = args.id
+
+    if ids and kill_all:
+        print(
+            "inductiva tasks kill: error: "
+            "argument id not allowed with argument --all",
+            file=sys.stderr)
+        return 1
+
+    if kill_all:
+        all_ids = []
+        for status in constants.TASK_RUNNING_STATUSES:
+            all_ids.extend(get_all(status=status))
+        ids = all_ids
 
-    for task_id in args.id:
-        inductiva.tasks.Task(task_id).kill(wait_timeout=args.wait_timeout)
+    confirm = args.yes or user_confirmation_prompt(
+        ids, __("task-prompt-kill-all"), __("task-prompt-kill-big", len(ids)),
+        __("task-prompt-kill-small"), kill_all)
```

Comment:
please consider readability.

Example:

```suggestion
    confirm = args.yes or \
        user_confirmation_prompt(ids,
                                 __("task-prompt-kill-all"),
                                 __("task-prompt-kill-big", len(ids)),
                                 __("task-prompt-kill-small"),
                                 kill_all)
    if not confirm:
        return 1
```